package.json

```diff
@@ -3,53 +3,51 @@
   "version": "0.0.0",
   "private": true,
   "license": "Apache-2.0",
-  "packageManager": "yarn@1.22.19",
-  "workspaces": {
-    "packages": [
-      "packages/*"
-    ],
-    "nohoist": [
-      "**/@types/**",
-      "**/typedoc",
-      "**/typedoc-plugin-markdown",
-      "**/typedoc-plugin-mdn-links"
-    ]
-  },
+  "packageManager": "pnpm@7.24.2",
   "dependencies": {
-    "lerna": "3.16.4"
+    "lerna": "5.6.2"
   },
   "devDependencies": {
+    "@types/node": "14.18.33",
     "@typescript-eslint/eslint-plugin": "5.21.0",
     "@typescript-eslint/parser": "5.21.0",
+    "@vercel/build-utils": "workspace:5.7.5",
+    "@vercel/fun": "1.0.4",
+    "@vercel/ncc": "0.24.0",
+    "@vercel/next": "workspace:3.3.9",
```

Comment:
Curious why `@vercel/next` is here. Where is it used?

Reply:
Ahh this ones tricky. The `@vercel/next` test suite invokes the `test/lib/run-build-lambda.js` which does some funky `require(build.use)` call. `build.use` returns `'@vercel/next'` and so it has to be included in the root devDep list

Reply:
Oh interesting.

@ijjk Perhaps we should have this script change directory into the package running it or update `use` to be the path to the current package.

Reply:
this modification feels like scope creep to me. its more of a "update `@vercel/next` tests" kinda change

Reply:
You're right. I just noticed its use for both `@vercel/next` and `@vercel/static-build` so lets save it for a future PR.